jenkins/src/cgcloud/jenkins/toil_jenkins_slave.py

```diff
@@ -56,7 +77,7 @@ def __disable_mesos_daemons( self ):
 
     @fabric_task
     def __install_parasol( self ):
-        run( "git clone git@github.com:BD2KGenomics/parasol-binaries.git" )
+        run( "git clone https://github.com/BD2KGenomics/parasol-binaries.git" )
```

Comment:
```
Arjuns-MacBook-Pro:~ arjun$ git clone git@github.com:BD2KGenomics/parasol-binaries.git
Cloning into 'parasol-binaries'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Arjuns-MacBook-Pro:~ arjun$ git clone https://github.com/BD2KGenomics/parasol-binaries.git
Cloning into 'parasol-binaries'...
remote: Counting objects: 24, done.
remote: Total 24 (delta 0), reused 0 (delta 0), pack-reused 24
Unpacking objects: 100% (24/24), done.
Checking connectivity... done.
Arjuns-MacBook-Pro:~ arjun$
```

I thought this might be an artifact of the recent overhaul github went through and made the change. The command didn't run on my laptop either. Maybe my git isn't set up right?